docs/core/compatibility/core-libraries/7.0/memorycache-tracking.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Breaking change: Tracking linked cache entries"
+description: Learn about the .NET 7 breaking change in .NET where MemoryCache no longer tracks linked cache entries by default.
+ms.date: 10/13/2022
+---
+# Tracking linked cache entries
+
+<xref:System.Runtime.Caching.MemoryCache> tracks linked cache entries to that options are propagated. In the following example, the `expirationToken` added for `child` is also applied to `parent`:
```

Comment:
I think you meant:

```suggestion
<xref:System.Runtime.Caching.MemoryCache> tracks linked cache entries so that options are propagated. In the following example, the `expirationToken` added for `child` is also applied to `parent`:
```